src/execution_plan/ops/op_cond_var_len_traverse.h

```diff
@@ -24,6 +24,8 @@ typedef struct {
 	bool expandInto;                /* Both src and dest already resolved. */
 	unsigned int minHops;           /* Maximum number of hops to perform. */
 	unsigned int maxHops;           /* Maximum number of hops to perform. */
+	NodeID dest_label_id;           /* ID of destination node label if known. */
+	const char *dest_label;         /* Label of destination node if known. */
```

Comment:
I believe these are unnecessary, see following example:
```
GRAPH.EXPLAIN g "match (a:S)-[:R *3..4]->(z:Z) return z"
1) "Results"
2) "    Project"
3) "        Conditional Traverse | (z:Z)->(z:Z)"
4) "            Conditional Variable Length Traverse | (a:S)-[anon_0:R*3..4]->(z:Z)"
5) "                Node By Label Scan | (a:S)"
```
dest node label should be set in `Conditional Traverse`

Reply:
Oh, good catch! Reverting.